sycl/include/sycl/info/aspects_deprecated.def

```diff
@@ -2,3 +2,4 @@ __SYCL_ASPECT_DEPRECATED(int64_base_atomics, 7, "use atomic64 instead")
 __SYCL_ASPECT_DEPRECATED(int64_extended_atomics, 8, "use atomic64 instead")
 // Special macro for aspects that don't have own token
 __SYCL_ASPECT_DEPRECATED_ALIAS(usm_system_allocator, usm_system_allocations, "use usm_system_allocations instead")
+__SYCL_ASPECT_DEPRECATED(host, 0, "use cpu instead")
```

Comment:
I'm not entirely sure if `cpu` is a replacement for `host`. Probably better just say that it is deprecated, because host device is removed from SYCL 2020